services/src/main/java/org/keycloak/services/resources/admin/AdminEventBuilder.java

```diff
@@ -250,6 +252,11 @@ public AdminEventBuilder representation(Object value) {
         if (value == null || value.equals("")) {
             return this;
         }
+
+        if (value instanceof UserRepresentation) {
+            StripSecretsUtils.strip((UserRepresentation) value);
```

Comment:
Not ideal to change the reference (remove credentials) here as it might not be the intention of the caller when calling this method.

But better than replicate `strip` in the difference places where the admin event is created.